### PR TITLE
feat(api): add import / export for generators

### PR DESCRIPTION
**Pull request by kuba-sewerynski**
# Pull Request

## Changes

Briefly describe your changes here.

## Why this change?

Explain the reason for the change.

## Testing

How was the change tested?

## Additional Notes

Any additional information or context you want to provide?

### CODE REVIEW

File: mostlyai/generators.py [hunk 1]
```diff
@@ -75,7 +75,44 @@ def create(self, config: dict) -> Generator:
...
+        self,
+        file_path: Union[str, Path],
+    ) -> Generator:
+        response = self.request(
+            verb=POST,
+            path=["import-from-file"],
+            headers={
+                "Content-Type": "multipart/form-data",
+                "Accept": "application/json, text/plain, */*",
+            },
```
Comment (michdr) [inline mostlyai/generators.py:88]: I'd extract this repeating part into some `_import_export_headers()` method

File: mostlyai/generators.py [hunk 2]
```diff
@@ -75,7 +75,44 @@ def create(self, config: dict) -> Generator:
         )
         return generator
 
+    def import_from_file(
```
Comment (michdr) [inline mostlyai/generators.py:78]: I guess it's coming up in the next commit: `_import_from_file` in `model.py:Generator` (same in `tools/...`)

Comment (kuba-sewerynski) [reply mostlyai/generators.py:79]: import_from_file method is not going to be executed on a specific generator, but one level higher

**Comment by shuangwu5** [comment]
`BaseModel.jinja2` was not updated. So I guess `make gen-public-model` was not run and `mostlyai/model.py` was updated manually?

**Comment by shuangwu5** [comment]
Please update the PR title to follow commitizen's style (e.g., starting with `feat(api): `), so that it can be shown in the CHANGELOG later.

**Comment by mplatzer** [comment]
> gen-public-model

@shuangwu5 this was now properly executed

**Pull request merged**